client/me/purchases/purchases-list/test/purchase-list-concierge-banner.tsx

```diff
@@ -76,16 +78,23 @@ describe( 'PurchaseListConciergeBanner tests', () => {
 				/>
 			</Provider>
 		);
-		expect( screen.queryByText( 'Your session is coming up soon' ) ).toBeInTheDocument();
+		expect(
+			screen.queryByText( 'Your Quick Start session is coming up soon' )
+		).toBeInTheDocument();
```

Comment:
Question: do we want to verify that the right link is being shown as well?

Reply:
I've added the URL checks